src/lamp_py/postgres/postgres_utils.py

```diff
@@ -581,8 +583,17 @@ def _rds_writer_process(metadata_queue: Queue[Optional[str]]) -> None:
         if metadata_path is None:
             break
 
-        insert_statement = sa.insert(MetadataLog.__table__).values(
-            path=metadata_path
+        insert_statement = (
+            postgresql.insert(MetadataLog.__table__)
+            .values(path=metadata_path)
+            .on_conflict_do_update(
+                index_elements=[MetadataLog.path],
+                set_={
+                    "rail_pm_processed": sa.false(),
+                    "rail_pm_process_fail": sa.false(),
+                    "created_on": now(),
+                },
```

Comment:
If INSERT fails, update process flags to false, so file will be re-loaded into performance manager database.